gitlab-pipeline/stage/post.yml

```diff
@@ -43,3 +43,40 @@ coveralls:finish-build:
   script:
     - 'curl -k ${COVERALLS_WEBHOOK_URL}?repo_token=${MENDER_COVERALLS_TOKEN} -d "payload[build_num]=$(git rev-parse HEAD)&payload[status]=done"'
 
+.trigger_template:
+  stage: .post
+  rules:
+    # - if: '$INTEGRATION_REV =~ /^[0-9]+\.[0-9]+\.[0-9]+(-build\d+)?$/'
+    - when: always
+    # the following is to prevent an endless loop of qa pipelines caused by downstream pipelines
+    - if: '$CI_PIPELINE_SOURCE == "pipeline"'
+      when: never
+
+trigger:mender-convert:
+  extends: .trigger_template
+  variables:
+    MENDER_ARTIFACT_VERSION: $MENDER_ARTIFACT_REV
+    MENDER_CLIENT_VERSION: $MENDER_REV
+    MENDER_ADDON_CONNECT_VERSION: $MENDER_CONNECT_REV
+    MENDER_ADDON_CONFIGURE_VERSION: $DEVICECONFIG_REV
```

Comment:
This is correct.

It also opens the Pandora box: we assume that at this point we have published the deb packages, which happens based on tagging the repos and not in control of mender-qa.

I suggest we use it as-is for the upcoming release, and study moving the tags management to `mender-qa` pipeline as the next step (maybe for the next release).

In any case, what we have introduced now is already very valuable because if the [non visible yet] tag based triggers for mender-dist-packages fail, the downstream pipeline here will fail making the release coordinator aware that something went wrong.

Reply:
**Discussed in a call**

We decided to address this in the current PR. So the expected outcome now is that we move the tags management to mender-qa (as it should have been in the first place), introducing then a mender-qa -> mender-dists-packages trigger.